Tasks/ANT/task.json

```diff
@@ -110,7 +110,7 @@
             "defaultValue": ".",
             "required": true,
             "groupName": "codeCoverage",
-            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. For example, classes1/Tests,classes2/Tests2. Refer http://www.eclemma.org/jacoco/trunk/doc/ant.html.",
+            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. This path is relative to the location of ANT build.xml directory. For example, classes1/Tests,classes2/Tests2. Refer http://www.eclemma.org/jacoco/trunk/doc/ant.html for Jacoco and https://github.com/cobertura/cobertura/wiki/Ant-Task-Reference for Cobertura.",
```

Comment:
Refer to the help text of Ant build file, Gradle Wrapper, Maven POM file fields of respective tasks for relative path